test/WebJobs.Extensions.DurableTask.Analyzers.Test/ActivityFunction/FunctionReturnTypeAnalyzerTests.cs

```diff
@@ -100,14 +104,20 @@ public static Object SayHello([ActivityTrigger] IDurableActivityContext context)
             return tuple;
         }
 
+        [FunctionName(""E1_SayHello_ValueTuple"")]
+        public static ValueTuple<string, int> SayHelloTuple([ActivityTrigger] ValueTuple<string, int> tuple)
+        {
+            return tuple;
+        }
+
         [FunctionName(""E1_SayHello_ReturnsString"")]
         public static string SayHelloDirectInput([ActivityTrigger] string name)
         {
             return $""Hello {name}!"";
         }
 
         [FunctionName(""E1_SayHello_ArrayAndNamedType"")]
-        public static List<string> SayHelloTuple([ActivityTrigger] Tuple<string, int> tuple)
+        public static List<string> SayHelloTuple([ActivityTrigger] ValueTuple<string, int> tuple)
```

Comment:
small nit: It may be worthwhile naming the method name of our functions what their function names currently are. Then we could use `nameof()` to reference it in more locations and make name changes easier to manage.

Reply:
Unfortunately, I can't use nameof() here to help manage test code changes because this is just one big string representing a document to be scanned by my analyzer. I can go through my test code and make sure the method names match the function names, though, just to clean it up a bit.

Reply:
I didn't realize from the GitHub page that this was just a raw string. This seems difficult to maintain.

I'm curious if we could make this code as a SampleFunctions.cs, and then have the test read the contents of that .cs file as a string. This would allow you to make use of intellisense and other important dev features for maintaining these samples, while still working within your test framework.

Reply:
Good suggestion. However, for every test case, there would have to be a cs file. I don't think it would improve much. However, I do think that these tests are very hard to read/understand and hard to add to and review, so I decided to make some changes to them. It will be much easier to identify the test cases now. I made these changes only for the function analyzer tests, for now, keeping the PR on theme with function analyzers. In the future, I will make changes to the rest of the tests to match.